### ISSUE TITLE
OF, CMake and OSX

### ISSUE THREAD

**Issue by symbolix** [BildPeter/ofxCMake#1]
Hi,
I would like to implement the CMake approach with OF. I am on OSX, using Vim. There are couple of resources lingering around, but most of them are too complicated and possibly outdated. I have seen your project and I am curios to know about your approach. I am new to OF and I am not very experienced with CMake but I am keen to get this working. Thanks.

**Comment by peterkolski** [comment]
Hey
This project was on hold till now, but I will be working on it now again. Your question is pretty vage, but I will add a README now, so it is usable. In principle it it already works for OSX. If you like to, test it out, and let me know about the results and issues.
All the best
Peter

**Comment by symbolix** [comment]
Hi Peter,
Thanks for replying me. I have built my own CMake workflow in the meantime and successfully compiled oF libs (static/shared) and also compiled couple the examples using pure CMake. I have implemented my solution from [here](https://github.com/kureta/oF-CLion). However, I am curious to test your approach as well. Will let you know (might take a while as I am working on another project.) Cheers!
